### PR TITLE
expose two new params for nanocompore

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
Ok, found another thing, do not merge yet, please.

File: tools/nanocompore/nanocompore_db.xml [hunk 1]
```diff
@@ -1,61 +1,56 @@
...
     <description>Process SampComp results database</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <expand macro="requirements"/>
     <command detect_errors="exit_code"><![CDATA[
-mkdir results && 
-tar -xf $db --strip-components 1 -C results/ && 
+mkdir -p results plots && 
+tar -xf $db --strip-components 1 -C results/ &&
```
Comment (nsoranzo) [inline tools/nanocompore/nanocompore_db.xml:9]:
```suggestion
tar -xf '$db' --strip-components 1 -C results/ &&
```

File: tools/nanocompore/nanocompore_db.xml [hunk 2]
```diff
@@ -1,61 +1,56 @@
...
+--out-dir ./plots/
 $bedgraph
+
     ]]></command>
     <inputs>
         <param name="reference" type="data" format="fasta" label="The reference genome used for read alignment."/>
         <param name="annotation" format="bed" type="data" label="BED file containing the annotation of the transcriptome used as reference when mapping"/>
         <param name="db" type="data" format="tar" label="SampComp Database" />
-        <param argument="--bedgraph" type="boolean" truevalue="--bedgraph" falsevalue="" label="Write output in BEDGRAPH format instead of BED"/>
+        <param argument="--pvalue-types" type="text" value="GMM_logit_pvalue,KS_dwell_pvalue,KS_intensity_pvalue" label="pvalue field names" help="Comma seperated values.">
```
Comment (nsoranzo) [inline tools/nanocompore/nanocompore_db.xml:24]:
```suggestion
        <param argument="--pvalue-types" type="text" value="GMM_logit_pvalue,KS_dwell_pvalue,KS_intensity_pvalue" label="pvalue field names" help="Comma-separated values.">
```

File: tools/nanocompore/NanocomporeDB_process.py [hunk 3]
```diff
@@ -52,9 +52,15 @@ def is_valid_directory(dir_name):
...
+    if args.bedgraph:
+        file_ext = 'bedgraph'
+    else:
+        file_ext = 'bed'
+
     if args.annotation_bed:
         for pt in args.pvalue_types.split(','):
-            print("bedgraph output for p-value type:", pt)
-            db.save_to_bed(output_fn='{}/{}.bedgraph'.format(args.out_dir, pt),
+            out_path = '{}/{}.{}'.format(args.out_dir, pt, file_ext)
```
Comment (nsoranzo) [inline tools/nanocompore/NanocomporeDB_process.py:62]:
```suggestion
            out_path = os.path.join(args.out_dir, '{}.{}'.format(pt, file_ext))
```

File: tools/nanocompore/nanocompore_db.xml [hunk 4]
```diff
@@ -1,61 +1,56 @@
...
         <test>
             <param name="reference" value="reference.fa"/>
             <param name="annotation" value="annot.bed"/>
-            <param name="db" value="DB.tar"/>
-            <output name="GMM_logit" file="GMM_logit_pvalue.bed"/>
-            <output name="KS_dwell" file="KS_dwell_pvalue.bed"/>
-            <output name="KS_intensity" file="KS_intensity_pvalue.bed"/>
+            <param name="db" value="DB.tar" ftype="tar" />
+            <output_collection name="pvalue_output" type="list" count="3">
+                <element name="GMM_logit_pvalue" file="GMM_logit_pvalue.bed" ftype="bed"/>
```
Comment (nsoranzo) [inline tools/nanocompore/nanocompore_db.xml:41]: Why are the `KS_dwell_pvalue.bed` and `KS_intensity_pvalue.bed` not checked any more?

Comment (bgruening) [reply tools/nanocompore/nanocompore_db.xml:41]: added them back! Thanks.

**Pull request merged**